src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,62 @@
+/* eslint-disable max-len */
 import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+import { Movie } from '../../types/Movie';
+
+interface CardProps {
```

Comment:
```suggestion
interface MovieCardProps {
```

lets make it more decsriptive